frontend/src/App.vue

```diff
@@ -437,16 +438,24 @@ export default {
     },
     toggleConsole() {
       this.showConsole = !this.showConsole;
+    },
+    init() {
+      this.yeomanName = "<no generator selected>";
+      this.promptIndex = 0;
+      this.prompts = [];
```

Comment:
this one is already defined in data. No need to do it again

Reply:
Must-to do it agian. here. next time data() is called ia after invoking 'receiveIsWebviewReady'

Reply:
Ok, i understand. No problem